### PR TITLE
Remove LikedMap event

### PR DESCRIPTION
**Pull request by mayoral**
This PR implements #12404.

### CODE REVIEW

File: app/controllers/carto/api/visualizations_controller.rb [hunk 1]
```diff
@@ -128,7 +128,6 @@ def add_like
         end
 
         event_properties = { user_id: current_viewer_id, visualization_id: @visualization.id, action: 'like' }
```
Comment (juanignaciosl) [inline app/controllers/carto/api/visualizations_controller.rb]: You kept `event_properties`, which is unused now.

File: app/controllers/carto/api/visualizations_controller.rb [hunk 2]
```diff
@@ -145,7 +144,6 @@ def remove_like
         @visualization.remove_like_from(current_viewer_id)
 
         event_properties = { user_id: current_viewer_id, visualization_id: @visualization.id, action: 'remove' }
```
Comment (juanignaciosl) [inline app/controllers/carto/api/visualizations_controller.rb]: Same.

File: app/controllers/carto/api/visualizations_controller.rb (hunk 1 above)
Comment (mayoral) [inline app/controllers/carto/api/visualizations_controller.rb]: Removed. Thanks for pointing that! :)

File: app/controllers/carto/api/visualizations_controller.rb (hunk 2 above)
Comment (mayoral) [inline app/controllers/carto/api/visualizations_controller.rb]: Same removal :P

**Pull request merged**